tests/fxl-page-spread_center/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>When rendition:page-spread-center is applied to a spine item, it should be displayed in the center of the screen and not in a synthetic spread.</dc:description>
```

Comment:
What is "it"? The page? The document? (I genuinely don't know. I would normally say "document," but can you have a multi-page document in FXL? I assume not so "page" is probably better.)